app/services/uphold/models/transaction.rb

```diff
@@ -28,9 +28,28 @@ def initialize(params = {})
       def all(id:)
         Rails.logger.info("Connection #{@uphold_connection.id} is missing uphold_access_parameters") and return if @uphold_connection.uphold_access_parameters.blank?
 
-        response = get(PATH.expand(id: id), {}, authorization(@uphold_connection))
+        index = 1
```

Comment:
Prefer to rename this to `page_index`

Reply:
https://github.com/brave-intl/publishers/pull/2330/commits/a6d89d6d16ca96edacdbb20bbfc71019dd3f54bd